mantidproject/mantid
# D22B new nexus

**Description of work.**
Adjusts the `LoadILLSANS` to adapt to the actual structure of the new real data.



**To test:**
- [ ] Make sure you can load numors 000101 to 000195 in 211/d22/internalUse.
- [ ] For a file of your choice, make sure that:

> The Z coordinate of the center bank center corresponds to `Detector 2.det2_calc` log value
> The Z coordinate of the right bank center corresponds to `Detector 1.det1_calc` log value
> The X coordinate of the right bank center corresponds to `Detector 1.dtr1_actual` log value (note the sign and unit in mm)
> L2 log should correspond to the distance of the center bank
> For both banks, the detector IDs should start in the bottom left corner and increment first upwards then rightwards (looking from sample, i.e Z-)
> The tubes must be masked vertically in both banks
> Check the tilt of the front bank to match `Detector 1.dan1_actual` log value



*There is no associated issue.*

*This does not require release notes because support for this instrument is added in this release.*

*Merge with the linked PR in geometry: https://github.com/mantidproject/mantidgeometry/pull/178* 



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
